voltdb/datadog_checks/voltdb/check.py

```diff
@@ -0,0 +1,148 @@
+# (C) Datadog, Inc. 2020-present
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+from typing import Any, List, Optional, cast
+
+import pkg_resources
+import requests
+from six import raise_from
+
+from datadog_checks.base import AgentCheck
+from datadog_checks.base.utils.db import Query, QueryManager
+
+from . import queries
+from .config import Config
+from .types import Instance
+
+BASE_PARSED_VERSION = pkg_resources.get_distribution('datadog-checks-base').parsed_version
+
+
+class VoltDBCheck(AgentCheck):
+    __NAMESPACE__ = 'voltdb'
+
+    def __init__(self, *args, **kwargs):
```

Comment:
```suggestion
    def __init__(self, name, init_config, instances):
```
Any particular reason for not using the typical signature?